Strategies/README.md

```diff
@@ -0,0 +1,23 @@
+# Strategies
+
+Learning strategies for [individuals](Individual.md), [teams](Team.md), and [communities](Community.md).
+
+These categories do not have strict boundaries between them: they are all complimentary and overlap in many places. We just use separate files to make it easier to find and reference them.
+
+## Why Strategies
+
+At Learners Guild, we are co-building an environment that is highly conducive to learning and growth.
+
+This is not easy.
+
```

Comment:
We are used to authority being centralized. When authority is distributed, it means every participant has more power, and with it, more decisions and more responsibility. 

In the same way, a citizen in a democracy has to be better informed and make more decisions than someone in a kingdom: A member of a learning collective carries more weight and autonomy than a student in a traditional educational institution.